sql/2020/09_Performance/lh5_performance_score_distribution.sql

```diff
@@ -0,0 +1,14 @@
+#standardSQL
+# Distribution of LH5 performance score.
+
+SELECT
+  SAFE_DIVIDE(slow,    slow + avg + fast + hundred) slow,
+  SAFE_DIVIDE(avg,     slow + avg + fast + hundred) avg,
+  SAFE_DIVIDE(fast,    slow + avg + fast + hundred) fast,
+  SAFE_DIVIDE(hundred, slow + avg + fast + hundred) hundred,
+FROM (
+  SELECT COUNTIF(score < 0.3) slow, COUNTIF(score > 0.3 and score < 0.8) avg, COUNTIF(score > 0.8 and score < 1.0) fast, COUNTIF(score = 1.0) hundred
+  FROM (
+    SELECT CAST(JSON_EXTRACT(report, '$.categories.performance.score') AS NUMERIC) score FROM `httparchive.lighthouse.2020_05_01_mobile`
```

Comment:
@thefoxis  Should we go back to the 2019 Almanac crawl to get LH5 data?
```suggestion
    SELECT
      CAST(JSON_EXTRACT(report, '$.categories.performance.score') AS NUMERIC) AS score
    FROM
      `httparchive.lighthouse.2019_07_01_mobile`
```

Reply:
We have similar query for LH6. Just wondering if the data set (list of URLs) would be the same if go back to 2019?

Reply:
The URLs change monthly, so imperfect coverage either way. AFAIK @thefoxis's reasoning for comparing LH versions was because LH was v5 during the 2019 Almanac, so this analysis is effectively a year-over-year comparison, which we could continue to do in future editions of this chapter.

Reply:
@thefoxis any preference for May 2020  or July 2019 for LHv5 data?

Reply:
@rviscomi no strong opinions here, whatever you think makes most sense going onwards (since you mentioned you'd do this in future chapters too). whatever you think is best!